crates/cubecl-linalg/src/matmul/components/global/load/strategy/async_full_cyclic.rs

```diff
@@ -39,15 +42,34 @@ impl<T: TilingOrder> LoadingValidation for AsyncFullCyclicLoading<T> {
 #[cube]
 impl<T: TilingOrder> AsyncFullLoadingStrategy for AsyncFullCyclicLoading<T> {
     type TilingLayout = ContiguousTilingLayout<T>;
+    type Job<MP: MatmulPrecision, CM: CopyMechanism<MP::ES>> = AsyncFullCyclicJob<MP, CM, T>;
 
-    fn load_full<MP: MatmulPrecision, G: GlobalConfig, CM: CopyMechanism<MP::ES>>(
-        read_view: &TensorReader<MP::EI>,
-        stage: &mut Stage<MP::ES, Self::TilingLayout>,
-        mechanism: &CM,
-        _quantization: CubeOption<Quantization<MP>>,
+    fn load_full<MP: MatmulPrecision, CM: CopyMechanism<MP::ES>, G: GlobalConfig>(
+        read_view: TensorReader<MP::EI>,
+        stage: Stage<MP::ES, Self::TilingLayout>,
+        mechanism: CM,
+        quantization: CubeOption<Quantization<MP>>,
         #[comptime] input_ident: InputIdent,
         #[comptime] config: G,
     ) {
+        default_async_full_load::<Self, MP, G, CM>(
+            read_view,
+            stage,
+            mechanism,
+            quantization,
+            input_ident,
+            config,
+        )
+    }
```

Comment:
Maybe `default_async_full_load` doesn't need quantization. If this will never be supported, we can add a 

```rust
comptime! {
  if quantization.is_some() {
    panic!("...") // or unimplemented!()
  }
}